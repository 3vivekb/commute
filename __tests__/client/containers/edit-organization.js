/* global describe, expect, it */

import { mount } from 'enzyme'
import { mountToJson } from 'enzyme-to-json'
import React from 'react'
import { Provider } from 'react-redux'

import { makeMockStore, mockStoreData } from '../../test-utils/mock-store.js'

import EditOrganization from '../../../client/containers/edit-organization'

const mockStore = makeMockStore(mockStoreData)

describe('Container > EditOrganization', () => {
  it('renders correctly', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <EditOrganization />
      </Provider>
    )
    expect(mountToJson(tree.find(EditOrganization))).toMatchSnapshot()
  })
})
