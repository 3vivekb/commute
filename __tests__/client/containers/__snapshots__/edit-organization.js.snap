exports[`Container > EditOrganization renders correctly 1`] = `
<Connect(EditOrganization)>
  <EditOrganization
    dispatch={[Function]}>
    <h1>
      Edit Organization
    </h1>
  </EditOrganization>
</Connect(EditOrganization)>
`;
